crypto/pkcs12/p12_sbag.c

```diff
@@ -103,6 +113,54 @@ PKCS12_SAFEBAG *PKCS12_SAFEBAG_create_crl(X509_CRL *crl)
                                     NID_x509Crl, NID_crlBag);
 }
 
+PKCS12_SAFEBAG *PKCS12_SAFEBAG_create_secret(int type, int vtype, const unsigned char *value, int len)
+{
+    PKCS12_BAGS *bag;
+    PKCS12_SAFEBAG *safebag;
+
+    if ((bag = PKCS12_BAGS_new()) == NULL) {
+        PKCS12err(PKCS12_F_PKCS12_SAFEBAG_CREATE_SECRET, ERR_R_MALLOC_FAILURE);
```

Comment:
Dont bother with flags anymore for new errors..
i.e use PKCS12err(0, ERR_R_MALLOC_FAILURE);